### ISSUE TITLE
Feature Request - Noise figure measurement with external noise source

### ISSUE THREAD

**Issue by Digitelektro** [erikkaashoek/NanoVNA#109]
Hello!
I am wondering if it would be feasible to implement noise figure measurement with an external noise source, like a HP 346B. I already tried to use it (with a tinySA Ultra), I made an excel sheet for calculating the device NF, the DUT NF & Gain, and I think I get reasonably good and persistent result. While I cannot compare it to a professional instrument, I have some minicircuits LNAs and other custom made LNAs and the result is in line with the specifications. 
It would be very helpful to have a menu where I enter the ENR value from the noise source, calibrate the tinySA and measure an LNA. What do you think?

**Comment by erikkaashoek** [comment]
Yes, this could work as you basically do a power level calibration using a noise source with known level and measure the difference in level with the noise source
But you can also check the noise marker by inputting the noise source and checking if the noise marker display the correct value.
